index.html

```diff
@@ -4,9 +4,46 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./client/assets/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <title>project!</title>
 
     <link href="./styles/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+
+  <body>
```

Comment:
Add comments to divide the different parts, for example:
```
<!-- hero -->
...
<!-- main -->
...
<!-- footer -->
...
```

Reply:
ok